spec_cleaner/rpmrequirestoken.py

```diff
@@ -21,6 +21,24 @@ def __init__(self, name, operator = None, version = None, prefix = None):
         self.operator = operator
         self.version = version
 
+    def _format_operator(self):
+        """
+        Make sure the operators look sane and not use all permutations
+        """
+        self.operator = self.operator.replace('=<', '<=')
+        self.operator = self.operator.replace('=>', '>=')
+
+    def _fix_pkgconfig_name(self):
+        # we just rename pkgconfig names to one unified one working everywhere
+        if self.name == 'pkgconfig(pkg-config)' or self.name == 'pkg-config':
+            self.name = 'pkgconfig'
+
+    def format_token(self):
+        if self.operator:
+            self._format_operator()
+        if self.name:
+            self._fix_pkgconfig_name()
```

Comment:
probably shouldn't be mutating method? either do this at init, or return a different instance which is fixed